front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/tags.ts

```diff
@@ -102,6 +103,20 @@ async function handler(
         });
       }
 
+      if (
+        !isBuilder(auth.getNonNullableWorkspace()) &&
+        (tagsToAdd.some((tag) => tag.reserved) ||
+          tagsToRemove.some((tag) => tag.reserved))
+      ) {
+        return apiError(req, res, {
+          status_code: 400,
+          api_error: {
+            type: "invalid_request_error",
+            message: "Reserved tags cannot be added or removed.",
```

Comment:
is this displayed to the user? Not sure this name make sense to be user facing 🙏🏻 
Maybe "protected" tag?